burn-no-std-tests/README.md

```diff
@@ -0,0 +1,26 @@
+The `burn-no-std-tests` contains integration tests aimed to check `no_std` compatibility of `burn`, `burn-core`, `burn-tensor` and `burn-ndarray` packages.
```

Comment:
I'm not sure we should create a package for that kind of tests, but instead building the examples without the `std` flag. It's not a decision we have to take right now, since this package will never be released, therefore easilly changed.

Reply:
OK. Let's brew this idea for a bit. Examples could serve as our integration tests. In fact, I wanted to have no_std example. Let's come back to this. I'll add TODO for this.